dist/js-reporters.js

```diff
@@ -0,0 +1,330 @@
+(function (global, factory) {
```

Comment:
Let's exclude "binaries" like this, we don't want to have multiple files change in every commit, even though we change just one thing. We can include this in releases.